src/pages/index.tsx

```diff
@@ -77,26 +73,30 @@ function renderCardList(tasks: task[], isEditable: boolean) {
       content={item}
       key={item.id}
       shouldEdit={isEditable}
-      onContentChange={async (id: string, newDetails: any) => isEditable
-        && updateCardContent(id, newDetails)}
+      onContentChange={async (id: string, newDetails: any) =>
+        isEditable && updateCardContent(id, newDetails)
+      }
     />
   ));
 }
 
 const Index: FC = () => {
+  const [editToggle, setEditToggle] = useState(false);
   const router = useRouter();
   const { query } = router;
   const [filteredTask, setFilteredTask] = useState<any>([]);
   const { response, error, isLoading } = useFetch(TASKS_URL);
-  const [IsUserAuthorized, setIsUserAuthorized] = useState(false);
-  const isEditable = !!query.edit && IsUserAuthorized;
+  const [IsUserAuthorized, setIsUserAuthorized] = useState(true);
+  const isEditable = !!query.edit && IsUserAuthorized && editToggle;
 
   useEffect(() => {
-    if ('tasks' in response) {
+    if ("tasks" in response) {
       const tasks = updateTasksStatus(response.tasks);
       tasks.sort((a: task, b: task) => +a.endsOn - +b.endsOn);
-      tasks.sort((a: task, b: task) => STATUS_ORDER.indexOf(a.status)
-        - STATUS_ORDER.indexOf(b.status));
+      tasks.sort(
```

Comment:
Why two sorts for `tasks`? can we do something so that lines no 95 and 96 can be clubbed?

Reply:
I didn't work on this. It is getting highlighted because of prettier. Correcting the settings